.github/workflows/ci.yml

```
@@ -84,3 +84,53 @@ jobs:
       # Soft-gate the audit with '|| true' so vulnerabilities are reported in the logs
       # but do not block pull request merges.
       - run: npm audit --audit-level=high --omit=dev || true
+
+  e2e-authenticated:
+    name: Authenticated E2E (Clerk)
+    runs-on: ubuntu-latest
+    # Skips (not fails) until CLERK_SECRET_KEY and NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY
+    # secrets are added — see #560. Never targets Preview/prod Supabase: this job
+    # boots a disposable local Supabase stack (Docker) and only writes there.
+    if: ${{ (secrets.CLERK_SECRET_KEY != '') && (secrets.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY != '') }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)